internal/contour/handler.go

```diff
@@ -146,19 +146,22 @@ func (e *EventHandler) Start(ctx context.Context) error {
 		// run to allow the holdoff timer to batch the updates from
 		// the API informers.
 		lastDAGRebuild = time.Now()
+
+		// initialSyncPollTimeout defines the duration to wait between polling attempts during the initial informer synchronization.
+		initialSyncPollTimeout = 100 * time.Millisecond
+
+		// initialSyncPollTimer is a timer set to trigger after the initialSyncPollTimeout duration.
+		initialSyncPollTimer = time.NewTimer(initialSyncPollTimeout)
```

Comment:
do we want a Timer or a Ticker here?

Reply:
I switched to using `Ticker`, as it seems a bit more elegant.

With `Timer` I had to call `Reset()` to restart timer after each poll if synchronization was not completed yet. With `Ticker` I need to call `Stop()` after the final attempt.